### ISSUE TITLE
Saved Search Dashlet - Admin layout issues

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/HPI#1715]
See the screencam below.  Issues I'm seeing:

- [x] In the saved search dashlet admin, the `Search Criteria` section is not displaying in the right.  Instead it's dropping below, outside of the well.
- [x] The Actions section is misaligned with the view that's generated when you click `Show`
- [x] Immediately after viewing the dashboard admin (no save needed), if the user navigates back to the dashboard, the actions do not appear.

![dashboard-admin](https://user-images.githubusercontent.com/1377228/41061498-70cff13c-6998-11e8-8784-2602f08f887c.gif)

**Comment by engerernoah** [comment]
HPI Revision: 20783

SAVED SEARCH DASHLET ADMIN LAYOUT- Reformatted the saved search dashlet layout on the admin page. The Search Criteria and Action Menu were outside of the view and the actions were misaligned from the action header. To fix this I moved the second column of the "Select configurable object types" section into the first row next to the header for that section. This allowed me to place the Search Criteria section underneath in its own row for better readability. To line up the Actions with the header I placed that into its own row underneath the Search Criteria Section. Layout and changes were run by Ian.

ACTION BUTTON NOT APPEARING AFTER VIEWING ADMIN PAGE- After viewing the saved search dashlet from the admin screen when pressing the button to show the actions the dashletActionConfig was being turned into a model, thus messing up the configuration to have it show on the dashboard. To fix this I made the dashletActionConfig a model in the initialize of savedSearchDashlet and reconfigured the files to grab the proper values from the model. 

CR: Rene and Joe
